action.yml

```
@@ -43,6 +43,10 @@ inputs:
   mappingFile:
     description: "The mapping.txt file used to de-obfuscate your stack traces from crash reports"
     required: false
+  changesNotSentForReview:
+    description: "The changesNotSentForReview option whether changes can be sent for review automatically or not"
+    required: false
+    default: 'false'
```

As you mentioned in the PR description, this is required to be true!!

Should this not be ?
```suggestion
    required: true
    default: 'true'
```

---

make scene, I think it should be `required: false` and `default: true` since we don't need to put default value when the options is required